native/cocos/platform/linux/LinuxPlatform.cpp

```diff
@@ -33,11 +33,17 @@
 #include "modules/Accelerometer.h"
 #include "modules/Battery.h"
 #include "modules/Network.h"
-#include "modules/Screen.h"
 #include "modules/System.h"
-#include "modules/SystemWindow.h"
 #include "modules/Vibrator.h"
 
+#if defined(CC_SERVER_MODE)
+    #include <cocos/platform/empty/modules/Screen.h>
+    #include <cocos/platform/empty/modules/SystemWindow.h>
```

Comment:
- It it any reason to use `<>` to include these header files?
- Do not need `cocos/`